api/nakadi-event-bus-api.yaml

```diff
@@ -1027,6 +1077,11 @@ definitions:
 
   EventTypeSchema:
     properties:
+      version:
+        type: string
+        description: |
+          Automatically generated value based on semantic versioning. Should not be modified by clients.
```

Comment:
Correct. Nakadi always generate a version for users. The specific rules are yet being developed but it will follow semver.